cmd/root.go

```diff
@@ -30,6 +31,7 @@ type args struct {
 	wantV6            bool
 	reverseDns        bool
 	verbose           bool
+	useDriver         bool
```

Comment:
I think it would be best to replace all variables from `UseDriver` to `UseWindowsDriver`.

It's slightly longer, but will be more readable, especially for people new to this repo and maybe our future self.